### ISSUE TITLE
Provide some way to use this tool without granting full github permissions

### ISSUE THREAD

**Issue by jyasskin** [w3c/ash-nazg#44]
For example, Googlers aren't allowed to give away write access, which this tool currently wants in order to auto-create repositories.

**Comment by dontcallmedom** [comment]
yeah, I find that quite annoying myself; I'll try to look into it.

**Comment by dontcallmedom** [comment]
Looking a bit into this, there are probably 3 levels of authentication that would be useful:
* no login at all required to see the list of managed repos, list of open / active pull requests
* full login required to access admin-like features (importing/creating repos/managing users)
* simple login (empty scope) required for personalized views, possibly for marking PRs as non-substantive (although this currently uses GH commenting which I don't think can be obtained without full write access to a repo)

@jyasskin for sake of clarity of what you want to achieve without full write access, are there specific features you would like to see or use in the repo manager?

**Comment by jyasskin** [comment]
@marcoscaceres was pushing this tool as the way to create repositories for new incubations. It turned out that I'd failed to communicate that we're having individuals create repositories that then move to the WICG, and so we need far more people to be able to create repos than he'd expected. He created https://github.com/WICG/starter-kit to serve that use case, so I'm not sure I still need this issue.
